src/platform_impl/macos/window.rs

```diff
@@ -1569,6 +1577,16 @@ impl WindowExtMacOS for UnownedWindow {
       ns_string_to_rust(tabbing_identifier)
     }
   }
+
+  #[inline]
+  fn set_traffic_light_position(&self, position: (f64, f64)) {
```

Comment:
```suggestion
  fn set_traffic_light_position<P: Into<Position>>(&self, position: P) {
```

Reply:
I don't really understand this. Do you want me to change only the public interface (i.e. set_traffic_..., with_...) or should I make the WindowBuilder field and the View attributes (or how they are called) also into a `position`? How do I extract x and y from this?

Reply:
This change should be added to all public APIs, and then you can convert them into logical position internally before applying the inset.

Reply:
Taking `Into<Position>` instead of `Into<LogicalPosition>` allows more flexibility and also to be consistent with other public APIs.

Reply:
The issue is that in the `set_traffic_light_position()` implementation I don't seem to have access to the `scale_factor()` method for `WindowBuilder` which is needed to convert from `Position` to `LogicalPosition`

Reply:
you can get the scale factor using `NSWindow::backingScaleFactor(ns_window)` like this https://github.com/tauri-apps/tao/blob/bd8bafeca8a063351022cd5d5791f3111e8761d0/src/platform_impl/macos/window.rs#L820

Reply:
Resolved?